README.md

```diff
@@ -39,4 +39,4 @@ Each platform maintains multiple types of releases:
 
 Copyright (c) 2018 SIL International. All rights reserved.
 
-Licensed under the [MIT](./windows/src/LICENSE.md) License.
\ No newline at end of file
+Licensed under the [MIT](./windows/src/LICENSE.md) License.
```

Comment:
This needs to stay but we need to list the exception for Linux at present I think.

Reply:
added "[KMFL](./linux) is licensed under GNU General Public License as published by the Free Software Foundation; either [version 2](./linux/libkmfl/COPYING) of the License, or (at your option) any later version."

Reply:
That was done for this PR then #870 does the change of licensing